metapool/sample_sheet.py

```diff
@@ -1488,42 +1524,99 @@ def __init__(self, path=None):
             }
 
 
-def load_sample_sheet(sample_sheet_path):
-    # Load the sample-sheet using various KLSampleSheet children and return
-    # the first instance that produces a valid sample-sheet. We assume that
-    # because of specific SheetType and SheetVersion values, no one sample
-    # sheet can match more than one KLSampleSheet child.
+def _parse_header(fp):
```

Comment:
It looks like this PR pulled in my recent changes to load_sample_sheet() and _parse_header(). Note these changes are already in the main branch.

Reply:
@AmandaBirmingham, I think a pull of the main branch from upstream in your PR will solve this.

Reply:
I did already pull from the main branch, which is why the changes that Charlie added to the main branch are in here ;)